modules/file-integrity-operator-release-channel.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="file-integrity-operator-release-channel_{context}"]
+= Configuring the File Integrity Operator release channel
+
+If you are upgrading an existing installation of File Integrity Operator to 1.0.0 or later, users must change the release channel to access the latest content.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+* You are subscribed to the `release-0.1` channel.
```

Comment:
should be `You are subscribed to the `v1` channel`?

Reply:
No, this is a prerequisite. So we are describing the requirements for the procedure. This procedure is specifically for customers that have FIO already installed and are using the `release-0.1` channel.